app/assets/javascripts/es6-components.js

```diff
@@ -0,0 +1,15 @@
+// These modules from govuk_publishing_components
+// depend on govuk-frontend modules. govuk-frontend
+// now targets browsers that support `type="module"`.
+//
+// To gracefully prevent execution of these scripts
+// on browsers that don't support ES6, this script
+// should be included in a `type="module"` script tag
+// which will ensure they are never loaded.
+
+//= require govuk_publishing_components/components/button
+//= require govuk_publishing_components/components/checkboxes
```

Comment:
I think `checkboxes` should stay where you previously had it, as it isn't on the list in [this document](https://docs.google.com/document/d/1uwip7pzQwM7t5ghn9_8KrsWXLePSRUltFPZ5fYw6Ab8/edit#heading=h.r2y7qlyvg991).

Reply:
Sorry it appears I missed it from the document, it should be included, I will update it now